dr_scaffold/scaffold_templates/view_templates.py

```diff
@@ -5,7 +5,59 @@
 VIEWSET = """class %(model)sViewSet(viewsets.ModelViewSet):
     queryset = %(model)s.objects.all()
     serializer_class = %(model)sSerializer
-    
+
+
+"""
+
+
+FULL_VIEWSET = """class %(model)sViewSet(
```

Comment:
Make use of model [ModelViewSet](https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset)

Reply:
I'm not a DRF expert but I thought that mixins would make it easy if someone would want to only have let's say a create and retrieve endpoints, I said he will remove other methods and mixins and only leave the ones he needs... is ModelViewSet broadly used than Mixins because I thought the opposite ? but would like to support all the widely used implementations and give the developer the ability to choose among them ... will need more discussion about these and will need to hear your thoughts!

Reply:
@elmkarami @yezz123 @rg3915 
A better idea would be to let the user pass a value of `CLRUD` to something like `--viewset` option. this will result to a full viewset class (as it is in the template), and a value of for example `CLR` will leave only the class with `create` and `list` and `retrieve` actions. 

The mechanism of selection I was thinking of is very simple let's split the ViewSet by "def" keyword then map to an array let's then map the last 5 items along with the keys "C", "L", "R", "U", "D" to a dictionary result will be something like :
```PYTHON
{
    "C":  "def create(self, request, *args, **kwargs):
        serializer = %(model)sSerializer(data=request.data)
        serializer.is_valid(raise_exception=True)
        serializer.save()
        return Response(serializer.data)",
    "L":  "def list(self, request, *args, **kwargs):
        queryset = self.get_queryset()
        serializer = %(model)sSerializer(queryset, many=True)
        return Response(serializer.data)",
    "R":  "def retrieve(self, request, *args, **kwargs):
        instance = self.get_object()
        serializer = %(model)sSerializer(instance=instance)
        return Response(serializer.data)",
     ...
}